finance/proposal-calls/cycle4/orgmycology.md

```diff
@@ -0,0 +1,122 @@
+- **1-2 - Special projects prioritized by CoCo**
+  - Contribution workshops - desing, support and test-out short workshops for PRs, contributing to documentation, learning about automated testing suites etc.
+  - How does Astropy want to work to intentionally welcome more newcomers?
+  - Website strategic facilitations - Drawing from community members vision of a more effective website to be used for a website revamp. 
+
+Across our three person team, we have the domain-specific (open science and OSS) experience and coaching skills to leave the organization in a strong position to get the most out of the community manager role, both in its current ½ time and future full-time capacity. We have extensive experience running interactive online workshops, designing surveys and interview schedules, constructing automations, and building active, participatory communities. We also understand and appreciate that community managers often become overwhelmed by the complex demands of their roles; all of our services are designed to alleviate this pressure by making it easier to gather and act upon feedback from the project community. 
+How does this benefit Astropy packages / users / community / ...? Short summary.
+
+
+Details here as needed for Astropy community members to understand what is planned.
+
+### Approximate Budget
+Currency: US $
+
+| Item                               | Cost                 |
+|------------------------------------|----------------------|
+| Community Manager Support          | $6,000               |
+| Qualitative Inquiry Coaching       | $3,000               |
+| Skills Map Development             | $12,000              |
+| Technical Skills Support           | $3,000               |
+| 1-2 CoCo prioritized projects      | $6,000-12,000 ($6,000 each) |
+| **Total**                          | **$30,000 - $36,000**|
+
+
+### Period of Performance
+
+6 Months starting April-September 2024 (could be readily extended if the community sees needs for this support longer-term)
+
+
+
+# OrgMycology Terms
+
+
+## Terms 
+
+A deposit of 30% of the fee is required to secure the dates and services. This is payable within 7 days of invoice. The balance will be invoiced at the agreed to monthly schedule of work.
+ 
+Cancellation Policy – 50% of our fees may be incurred for cancellation 10 to 5 working days from the date of a significant portion of work (e.g. facilitated workshop, extended meeting or design session).
+ 
+100% of our fees may be incurred for cancellation within 5 working days from the date of a significant piece of work.
+ 
+Any non-refundable expenses such as travel and accommodation that have already been booked will be charged in full.
+
+Travel (if applicable): Passed on at cost.  We are also happy for your team to book any travel or accommodation directly. Efforts will be made to share travel expenses with other clients, and to use Zoom/Teams where appropriate.
```

Comment:
Note that there are limitations to travel in the NASA grant.

Reply:
Can do, thats our standard consultant boilerplate. It is not applicable to this effort, so I will remove.